sdk/storage/storage-blob-changefeed/README.md

```diff
@@ -0,0 +1,158 @@
+
+## Examples
+
+### Initialize the change feed client
+
+The `BlobChangeFeedClient` requires a `BlobServiceClient` to initialize. Refer to [storage-blob](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-blob#create-the-blob-service-client) for how to create the blob service client. Here is an example using `StorageSharedKeyCredential`.
+
+  ```javascript
+  const { BlobServiceClient, StorageSharedKeyCredential } = require("@azure/storage-blob");
+  const { BlobChangeFeedClient } = require("@azure/storage-blob-changefeed");
+
+  // Enter your storage account name and shared key
+  const account = "<account>";
+  const accountKey = "<accountkey>";
+  // Use StorageSharedKeyCredential with storage account and account key
+  // StorageSharedKeyCredential is only available in Node.js runtime, not in browsers
+  const sharedKeyCredential = new StorageSharedKeyCredential(account, accountKey);
+  const blobServiceClient = new BlobServiceClient(
+    `https://${account}.blob.core.windows.net`,
+    sharedKeyCredential
+  );
+
+  const changeFeedClient = new BlobChangeFeedClient(blobServiceClient);
+  ```
+
+### Reading all events in the Change Feed
+
+Use `BlobChangeFeedClient.getChanges()` to get iterators to iterate through the change events.
+
+```javascript
+const { BlobChangeFeedEvent } = require("@azure/storage-blob-changefeed");
+
+let changeFeedEvents : BlobChangeFeedEvent[] = [];
+for await (const event of changeFeedClient.getChanges()) {
+    changeFeedEvents.push(event);
+}
+```
+
+By page.
+
+```javascript
+const { BlobChangeFeedEvent } = require("@azure/storage-blob-changefeed");
+
+let changeFeedEvents : BlobChangeFeedEvent[] = [];
+for await (const eventPage of changeFeedClient.getChanges().byPage()) {
+    for (const event of eventPage) {
+        changeFeedEvents.push(event);
+    }
+}
+```
+
+### Resuming reading events with a cursor
+
+```javascript
+const { BlobChangeFeedEvent } = require("@azure/storage-blob-changefeed");
+
+let changeFeedEvents : BlobChangeFeedEvent[] = [];
+const firstPage = await changeFeedClient.getChanges().byPage({maxPageSize: 10}).next();
+for (const event of firstPage) {
+    changeFeedEvents.push(event);
+}
+
+// Resume iterating from the pervious position with the continuationToken.
+for await (const eventPage of changeFeedClient.getChanges().byPage({continuationToken: firstPage.continuationToken})) {
+    for (const event of eventPage) {
+        changeFeedEvents.push(event);
+    }
+}
+```
+
+### Reading events within a time range
+
+Pass start time and end time to `BlobChangeFeedClient.getChanges()` to fetch events within a time range.
+
+Note that for this preview release, the change feed client will round start time down to the nearest hour, and round end time up to the next hour.
+
+```javascript
+const { BlobChangeFeedEvent } = require("@azure/storage-blob-changefeed");
+
+const start = new Date(Date.UTC(2020, 1, 21, 22, 30, 0)); // will be floor to 22:00
```

Comment:
I don't understand the "floor to"/"ciel to" comments, could this be clarified some?

Reply:
floor to: rounded down to the nearest hour
ceil to: rounded up to the nearest hour